### DISCUSSION TITLE
Unable to call azure durable function because of CORS error while authentication

### DISCUSSION THREAD

**Discussion by softwaretrackmood** [Azure/azure-functions-durable-extension#2483] [Q&A]
Unable to call durable function response StatusQueryGetUri because of cors error.

**Comment by FFranz93** [comment]
Can you give us some more information about how your application is set-up? 
On the first glance it looks like on your Function App CORS is not configured correctly.
You can do that in the Azure portal by going to your Function App and then in the left menu under "API" see "CORS".
Besides that I'd recommend that you anonymize some of the values in your screenshots, because they contain some information that might be sensitive.
